src/ssl.c

```diff
@@ -6153,9 +6154,146 @@ int wolfSSL_CertManagerDisableCRL(WOLFSSL_CERT_MANAGER* cm)
 
     return WOLFSSL_SUCCESS;
 }
+
+#ifndef NO_WOLFSSL_CM_VERIFY
+void wolfSSL_CertManagerSetVerify(WOLFSSL_CERT_MANAGER* cm, VerifyCallback vc)
+{
+    WOLFSSL_ENTER("wolfSSL_CertManagerSetVerify");
+    if (cm == NULL)
+        return;
+
+    cm->verifyCallback = vc;
+}
+
+static int CertManager_DoVerifyCallback(WOLFSSL_CERT_MANAGER* cm, int ret,
```

Comment:
In the ideal world this function should be combined with internal.c `DoVerifyCallback` to eliminate duplicate code. If you rebase with latest master the `DoVerifyCallback` function has been improved to not do the SSL alert, you may find its possible to combine these now.